src/SuperDumpService/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace SuperDumpService {
 	public static class Program {
 		public static void Main(string[] args) {
 			var host = new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(opt => opt.Limits.MaxRequestBodySize = 8589934592L)  // 8gb
```

Comment:
The only vulnerability I see is when people start flooding the server with huge requests (therefore maybe resulting in a DoS). I think we could live with that, seeing that we'd have to choose a very high limit anyway to allow dump files.